.github/workflows/positron-full-test.yml

```diff
@@ -146,6 +174,13 @@ jobs:
         id: electron-smoke-tests
         run: DISPLAY=:10 yarn smoketest-no-compile --tracing
 
+      - name: Run Web Smoke Tests
+        env:
+          POSITRON_PY_VER_SEL: 3.10.12
+          POSITRON_R_VER_SEL: 4.4.0
+        id: electron-web-smoke-tests
+        run: DISPLAY=:10 yarn smoketest-web --tracing
+
```

Comment:
For security's sake I'd have a step that runs at the end that removes the license file `pdol_rsa` when we're doing using it so it doesn't hang around on the runner after the test is finished (whether it succeeds or fails)

Reply:
Ok, will do!